sdk/communication/Azure.Communication.JobRouter/README.md

```diff
@@ -0,0 +1,298 @@
+# Azure Communication JobRouter client library for .NET
+
+This package contains a C# SDK for Azure Communication Services for JobRouter.
+
+[Source code][source] | [Package (NuGet)]<!-- [https://www.nuget.org/packages/Azure.Communication.JobRouter]--> | [Product documentation][product_docs]
```

Comment:
This renders incorrectly due to the comment.   I'd recommend routing this to the NuGet home page until you're published.    _(I'd also suggest a tracking issue to remember to set the actual link before you publish)_

Reply:
https://github.com/Azure/azure-sdk-for-net/issues/30319